modules/swagger-codegen/src/main/resources/swift/api.mustache

```diff
@@ -80,11 +80,14 @@ public class {{classname}}: APIBase {
         ]{{/hasMore}}{{/formParams}}{{/queryParams}}{{#queryParams}}{{^secondaryParam}}[{{/secondaryParam}}
             "{{baseName}}": {{paramName}}{{#isInteger}}{{^required}}?{{/required}}.encodeToJSON(){{/isInteger}}{{#isLong}}{{^required}}?{{/required}}.encodeToJSON(){{/isLong}}{{#hasMore}},{{/hasMore}}{{^hasMore}}
         ]{{/hasMore}}{{/queryParams}}
-        let parameters = APIHelper.rejectNil(nillableParameters){{/bodyParam}}
-
+ 
+        let parameters = APIHelper.rejectNil(nillableParameters)
+ 
+        let convertedParameters = APIHelper.convertBoolToString(parameters!){{/bodyParam}}
```

Comment:
Maybe it would make more sense to write convertBoolToString to return a non optional type so you don't have to force unwrap here?